File: AthleteApp/AthleteApp/PreviewData/MorningBriefingPreviewData.swift
import Foundation

enum MorningBriefingPreviewData {
    static let marcin = MorningBriefingPresentation(
        greeting: "Dzień dobry",
        athleteName: "Marcin",
        dateText: "Poniedziałek, 3 sierpnia",
        coachMessage: """
        Dzień zapowiada się bardzo dobrze.

        Po dwóch dniach regeneracji organizm jest gotowy na trening jakościowy.

        Największą korzyść przyniesie jakość, nie objętość.
        """,
        decision: .init(
            title: "Trening progowy",
            duration: "60–75 min",
            intensity: "Strefa 3–4"
        ),
        reasons: [
            "HRV wróciło do normy",
            "Sen był lepszy niż zwykle",
            "Zmęczenie spadło",
        ],
        planItems: [
            .init(id: "training", title: "Trening progowy", systemImage: "figure.run"),
            .init(id: "nutrition", title: "80 g węglowodanów", systemImage: "fork.knife"),
            .init(id: "sleep", title: "Sen przed 23:00", systemImage: "moon.stars.fill"),
        ],
        goal: .init(
            title: "Budowa wydolności",
            progress: 0.75,
            timeline: "Tydzień 3 z 12"
        )
    )
}

File: web/AthleteWeb/src/preview-data/morning-briefing-preview-data.ts
import type { MorningBriefingPresentation } from "../models/morning-briefing-presentation";

export const morningBriefingPreviewData: MorningBriefingPresentation = Object.freeze({
  greeting: "Dzień dobry",
  athleteName: "Marcin",
  dateText: "Poniedziałek, 3 sierpnia",
  timeText: "07:30",
  coachMessage: Object.freeze([
    "Dzień zapowiada się bardzo dobrze.",
    "Po dwóch dniach spokojniejszego treningu organizm dobrze się zregenerował.",
    "Dzisiaj warto wykonać trening progowy.",
    "Największą korzyść przyniesie jakość, nie objętość.",
  ]),
  decision: Object.freeze({
    title: "Trening progowy",
    duration: "60–75 min",
    intensity: "Strefa 3–4",
  }),
  reasons: Object.freeze([
    "HRV wróciło do normy",
    "Sen był lepszy niż zwykle",
    "Zmęczenie spadło",
  ]),
  changesSinceYesterday: Object.freeze([
    "HRV poprawiło się",
    "Sen był dłuższy",
    "Zmęczenie jest niższe",
  ]),
  todayPlan: Object.freeze([
    "Trening progowy",
    "80 g węglowodanów przed treningiem",
    "Sen przed 23:00",
  ]),
  goal: Object.freeze({
    title: "Budowa wydolności",
    progressLabel: "75%",
    progressValue: 0.75,
    timeline: "Tydzień 3 z 12",
  }),
  shortcuts: Object.freeze([
    Object.freeze({ id: "recovery", label: "Regeneracja" }),
    Object.freeze({ id: "training", label: "Trening" }),
    Object.freeze({ id: "nutrition", label: "Odżywianie" }),
    Object.freeze({ id: "history", label: "Historia" }),
  ]),
});
